.gitignore

```diff
@@ -1,2 +1,27 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
This change seems outside the scope of the PR.

Reply:
got it, check [revert .gitignore](https://github.com/joeyates/exiffer/pull/1/commits/46427f9d02da030323c7669186148f4e795edfd7)

Reply:
Maybe create a new PR for this?

Reply:
The .gitignore file in this project is "opinionated". It follows the idea that an Elixir developer will have a global .gitignore with the standard exclusions, plus those which are specific to their situation (e.g. `.DS_Store` if they use macOS). This libraries only specificity is that it actually generates documentation - not all projects do, so that is explicitly handled.